### ISSUE TITLE
traceback

### ISSUE THREAD

**Issue by bloedlink** [firecat53/carpool#1]
Traceback (most recent call last):
  File "C:\Users\niels\AppData\Local\Programs\Python\Python38-32\lib\site-packages\bottle.py", line 868, in _handle
    return route.call(**args)
  File "C:\Users\niels\AppData\Local\Programs\Python\Python38-32\lib\site-packages\bottle.py", line 1748, in wrapper
    rv = callback(*a, **ka)
  File "app.py", line 40, in index
    ns = c.execute("SELECT next_shift FROM carpool LIMIT 1").fetchone()[0]
TypeError: 'NoneType' object is not subscriptable
127.0.0.1 - - [08/Jan/2020 20:27:03] "GET / HTTP/1.1" 500 741

**Comment by bloedlink** [comment]
Sorry to bother, also very new to coding and finding my way into python and django right now. 
Would i be able to apply this in a django enviroment?

Kind regards

**Comment by firecat53** [comment]
No, this is specifically for bottle.py. Django is a much larger beast. This is an extremely simple single purpose webapp. You might work through the bottle tutorial and then see if you can get this working.
